fetcher/fetcher.go

```diff
@@ -124,7 +124,6 @@ func (bot *fetcherImpl) catchupLoop(ctx context.Context) error {
 			if ctx.Err() != nil {
 				return fmt.Errorf("catchupLoop() fetch err: %w", err)
 			}
-			bot.setError(err)
```

Comment:
So we just want to permanently not set the error?  Just confirming if this is intended functionality.

Reply:
Yes, my intention is to log errors returned from Algod client but not set them.

The error returned when the catchup loop has caught up is https://github.com/algorand/go-algorand-sdk/blob/develop/client/v2/common/common.go#L79 just a generic typedef for error, so as far as I can tell there is not a great way to differentiate between an aglod error that is expected because we caught up versus one this stems from a problem.

Reply:
If there was some other, more severe error, it would be triggered moments later in the regular block following routine.